client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -134,6 +143,7 @@ type PlanFeatures2023GridType = PlanFeatures2023GridProps &
 
 type PlanFeatures2023GridState = {
 	showPlansComparisonGrid: boolean;
+	selectedStorage: PlanSelectedStorage;
```

Comment:
Just want to reemphasize this a little. Our immediate next priority for the plans package is https://github.com/Automattic/wp-calypso/issues/78266, which is about breaking apart the grid components and exporting individually. This is my next issue on the queue.

So we need to keep our focus/priorities on the refactor we discussed previously on a few occassions (to remove the local `selectedStorage` state from `PlanFeatures2023Grid` and use the `wpcom-plan-ui` data store instead):

1. Either do it here, so we can test it already (and keep testing in follow-ups).
2. Do it immediately after, once the comparison grid needs access to these selections.

To say it differently, let's not propagate the local state variable `selectedStorage` to the comparison grid component at any point. It should access that concept from a data store - there will not be a shared state between comparison grid and feature grid going forward (we can assume there isn't one already).

Reply:
Yep. This was not forgotten. I'm still planning to do it in the immediate follow-up, so I'm opting for option #2. Just waiting on this PR to merge before continuing work since the underlying implementation has shifted so much https://github.com/Automattic/wp-calypso/pull/79104